scripts/environment.sh

```diff
@@ -64,7 +64,7 @@ export PATH=$PATH:$HOME/.krew/bin
 
 # Upgrade test environment vars and config
 if [[ ${GINKGO_FOCUS:-} == "clusterctl-upgrade" ]]; then
-  export NUM_NODES=${NUM_NODES:-"5"}
+  export NUM_NODES="5"
```

Comment:
why are we hardcoding it?

Reply:
for `clusterctl-upgrade` it is always 5. I think it's ok to hard-code.

Reply:
true but my point is this script is not used for clusterctl-upgrade only

Reply:
only when GINKGO_FOCUS=="clusterctl-upgrade" in that scenario we are hardcoding it to 5.

Reply:
true,

/approve